tmate/session@.service

```diff
@@ -5,9 +5,9 @@ PartOf=workers.target
 [Service]
 User=villager
 Type=simple
-ExecStartPre=/usr/bin/mkdir -P /tmp/devices/%i
+ExecStartPre=/bin/mkdir -p /tmp/devices/%i
 ExecStart=/usr/bin/tmate -F -S /tmp/devices/%i/%i.sock new-session
-EnvironmentFile=/root/%i
+EnvironmentFile=/root/device%i
```

Comment:
No this is wrong. the `%i` contains device in it. It is everything after the `@` in the service name up to the `.service`

Reply:
I did notice the change to only `%i` as opposed to `device%i` elsewhere. But `create_config.py` is writing the AUTH token to `/root/device%i` ,  not `/root/%i` . Without this change the tmate sessions do not get `$AUTH` set

Reply:
c.f. https://github.com/linted/HardwareCheckout/blob/dd5cdeee01ab727fdf63c74090e22b51aa907cac/tmate/create_config.py#L26

Reply:
oh oops -- I _just_ parsed what you said. OK I'll fix it